end-end-tests/api-standards/resources/thing/2021-11-10/002-fail-batch-post.yaml

```diff
@@ -0,0 +1,137 @@
+openapi: 3.0.3
+x-snyk-api-stability: experimental
+info:
+  title: v3
+  version: 3.0.0
+servers:
+  - url: https://api.snyk.io/v3
+    description: Public Snyk API
+tags:
+  - name: Thing
+    description: Short description of what Thing represents
+paths:
+  /orgs/{org_id}/thing:
+    post:
+      summary: Create a new thing
+      description: Create a new thing
+      operationId: createThing
+      tags:
+        - Thing
+      parameters:
+        - { $ref: "#/components/x-rest-common/parameters/Version" }
+        - { $ref: '#/components/parameters/OrgId' }
+      requestBody:
```

Comment:
@cmars  -- Fixed the test cases for this instance. 

Also realized along the way that none of the e2e tests (even those for regular `post`) have a `requestBody